porch/repository/pkg/git/git.go

```diff
@@ -211,12 +174,12 @@ func (r *gitRepository) ListPackageRevisions(ctx context.Context) ([]repository.
 }
 
 func (r *gitRepository) CreatePackageRevision(ctx context.Context, obj *v1alpha1.PackageRevision) (repository.PackageDraft, error) {
-	main, err := r.repo.Reference(refMain, true)
-	if err != nil {
-		return nil, err
+	var base plumbing.Hash
+	if main, err := r.repo.Reference(refMain, true); err == nil {
```

Comment:
Nit: I feel like checking the error is a good idea here, both because it _might_ be something serious (I'm guessing we're anticipating a "not found" type error).  But also checking for not-found also makes the code more self-documenting.

Reply:
ack, done.